File: KMTracker/KMTracker/Coordinate.cs
namespace KMTracker
{
	public class Coordinate
	{
		public Longitude Longitude;
		public Latitude Latitude;


		public Coordinate(Latitude latitude, Longitude longitude)
		{
			Longitude = longitude;
			Latitude = latitude;
		}

		public Coordinate(double v1, double v2)
		{
			Longitude = new Longitude { longitude = v1 };
			Latitude = new Latitude { latitude = v2 };
		}
	}
}

File: Server/api/models/Coordinate.js
/**
 * Coordinate.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {

  autoCreatedAt: false,
  autoUpdatedAt: false,

  attributes: {

    rit: {
      model: 'rit'
    },

    latitude: {
      type: 'string'
    },

    longitude: {
      type: 'string'
    }

  }
};

